### PR TITLE
Feature/render app in modal

### PR DESCRIPTION
**Pull request by Mihran9991**

### CODE REVIEW

File: src/renderer/renderer.ts [hunk 1]
```diff
@@ -147,6 +147,8 @@ class Renderer implements IRenderer {
     const frame = document.createElement('iframe');
     frame.style.width = '100%';
     frame.style.height = '100%';
+    frame.style.border = '1px solid #ccc';
```
Comment (shahen94) [inline src/renderer/renderer.ts:150]: This should be by default ?

Or this was made only for example/index.html ?

Comment (Mihran9991) [reply src/renderer/renderer.ts:150]: I think, this can be reverted!

Comment (shahen94) [reply src/renderer/renderer.ts]: Let's revert background color as well 🙏

Comment (Mihran9991) [reply src/renderer/renderer.ts]: Yep, no problem

**Pull request merged**